testing/tests/DevExpress.ui.widgets/treeViewParts/lazyRendering.js

```diff
@@ -22,6 +28,81 @@ QUnit.test("Only root nodes should be rendered by default", function(assert) {
     assert.equal(items.length, 2);
 });
 
+['!/#$%&\'()"+./:;<=>?@[]^`{|}~\\,', '____2______.jpg', 'E:\\test\\[gsdfgfd]  |  \'[some__file]', '!@#$%^&*()_+'].forEach((testId) => {
```

Comment:
[TreeView.keyExpr](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxTreeView/Configuration/#keyExpr) can refer to a property that returns a 'number' value. I think we need to add the Number(1) value.